Dockerfile

```diff
@@ -2,24 +2,24 @@ ARG BASE_IMAGE=docker.io/fedora:32
 
 FROM ${BASE_IMAGE}
 
-ADD demo/project /runner/project
-ADD demo/env /runner/env
-ADD demo/inventory /runner/inventory
-
-# UNDO Before 2.0 Release:
-# Install Ansible and Runner
-#ADD https://releases.ansible.com/ansible-runner/ansible-runner.el8.repo /etc/yum.repos.d/ansible-runner.repo
-#RUN dnf install -y ansible-runner
-RUN dnf install -y python3-pip rsync openssh-clients sshpass glibc-langpack-en git \
+# Install system packages for use in all images
+RUN dnf install -y \
+    python3-pip \
+    python3-devel \
+    gcc \
+    rsync \
+    openssh-clients \
+    sshpass \
+    glibc-langpack-en \
+    git \
```

Comment:
We might want to remove git from here now since it should live in the builder-side metadata.

Reply:
you're right, but in this case, the git module is part of ansible-base.

Philosophically I would prefer to remove it, but it would push it to an `append` step type customization in the definition. Also, I suspect it's needed by the `pip install git+...` lines too.

So I want to, but these are the reasons we haven't.

Reply:
Fair enough. Marging.